doc/api.md

```diff
@@ -1,14 +1,14 @@
-# API Reference
```

Comment:
These changes will break the docs. Are you using Windows by any chance?

Reply:
Yes, I use Windows. But I can do the npm run doc from Linux if need be